core/store/src/trie/trie_storage.rs

```diff
@@ -74,35 +74,38 @@ impl TrieStorage for TrieMemoryPartialStorage {
     }
 }
 
+/// Maximum number of cache entries.
+#[cfg(not(feature = "no_cache"))]
+const TRIE_MAX_CACHE_SIZE: usize = 10000;
+
+#[cfg(feature = "no_cache")]
+const TRIE_MAX_CACHE_SIZE: usize = 1;
+
+/// Values above this size are never cached.
+/// Note that Trie inner nodes are always smaller than this.
+const TRIE_MAX_VALUE_CACHED: usize = 4000;
```

Comment:
It's better to annotate the unit as well (looks like it's byte here).

Reply:
In the comment?

Reply:
yeah

Reply:
I suggest we call it `TRIE_LIMIT_CACHED_VALUE_SIZE`; I find the current name confusing as it sounds like there are values 1, 2, 3, .... 4000, 100500, and we only cache the values that are below 4000 (max value).